doc/watir-installation.md

```diff
@@ -0,0 +1,16 @@
+# chrome install for wsl
```

Comment:
Nice doc folder to keep the fighting path.

Reply:
Thanks haha